### PR TITLE
feat: read change requests for insights

### PR DESCRIPTION
**Pull request by kwasniew**
## About the changes

* introducing read model for project insights
* adding one method with change request counts
* removing unused code in project service

Next PR:
* add more tests with non zero values
* consider moving some of the code to the new store instead of coupling project insights service to stores and services from other subdomains

### CODE REVIEW

File: src/lib/features/project-insights/project-insights-read-model.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        };
+
+        const rows: Array<{ state: ChangeRequestDBState; count: string }> =
+            await this.db('change_requests')
+                .select('state')
+                .count('* as count')
+                .where('project', '=', projectId)
+                .groupBy('state');
+
+        return rows.reduce((acc, current) => {
```
Comment (kwasniew) [inline src/lib/features/project-insights/project-insights-read-model.ts:38]: I consider removing some duplication here in next PR

**Review by sjaanus — APPROVED** [review]
